package-lock.json

```diff
@@ -8089,20 +8089,20 @@
       }
     },
     "node_modules/@salesforce/source-deploy-retrieve": {
```

Comment:
I'm kind of amazed this compiles without syncing the apex/templates dependancies as well

Reply:
Should I submit a PR against templates to upgrade SDR to the same version in that repo also?

Reply:
Oh wait nm templates doesn't depend on SDR.